/jds/bin/diff -uprN rdesktop-1.6.0.old/doc/rdesktop.1 rdesktop-1.6.0/doc/rdesktop.1
--- rdesktop-1.6.0.old/doc/rdesktop.1	2009-02-20 16:33:19.955387000 +0000
+++ rdesktop-1.6.0/doc/rdesktop.1	2009-02-20 16:35:26.436521000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .TH rdesktop 1 "November 2005"
 .SH NAME
 .I rdesktop
@@ -220,9 +221,25 @@ Use RDP version 4.
 .BR "-5"
 Use RDP version 5 (default).
 .PP
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWrdesktop
+Interface stability^Volatile
+.TE
+.sp
 .SH LINKS
 Main website of rdesktop
 .br
 \fIhttp://www.rdesktop.org/
 .LP
 .PP
+...\" LSARC 2008/083 rdesktop
